test/NewsService.Data.UnitTests/NewsRepositoryTests.cs

```diff
@@ -18,12 +20,17 @@ class NewsRepositoryTests
         private IDataProvider _provider;
         private INewsRepository _repository;
 
-        private DbNews dbNewsRequest;
-        private DbNews dbNews;
-        private DbNews dbNewsToAdd;
+        private DbNews _dbNews;
+        private DbNews _dbNewsToAdd;
+        private JsonPatchDocument<DbNews> _editDbNews;
+        private JsonPatchDocument<DbNews> _editDbNewsSubject;
+        private JsonPatchDocument<DbNews> _editDbNewsContent;
+        private JsonPatchDocument<DbNews> _editDbNewsIsActive;
 
         private Guid _firstUserId = Guid.NewGuid();
         private Guid _secondUserId = Guid.NewGuid();
+        private string _firstValue;
```

Comment:
Assign these values, because now you are sending null and getting null

Reply:
Thanks, done